connect/runtime/src/main/java/org/apache/kafka/connect/runtime/Worker.java

```diff
@@ -259,6 +259,20 @@ public boolean isSinkConnector(String connName) {
      * @return a list of updated tasks properties.
      */
     public List<Map<String, String>> connectorTaskConfigs(String connName, int maxTasks, List<String> sinkTopics) {
+        return connectorTaskConfigs(connName, maxTasks, sinkTopics, null);
+    }
+
+    /**
+     * Get a list of updated task properties for the tasks of this connector.
+     *
+     * @param connName the connector name.
+     * @param maxTasks the maxinum number of tasks.
+     * @param sinkTopics a list of sink topics.
+     * @param topicsRegex a regex defining sink topics to include.
+     * @return a list of updated tasks properties.
+     */
+    public List<Map<String, String>> connectorTaskConfigs(String connName, int maxTasks, List<String> sinkTopics,
+                                                          String topicsRegex) {
```

Comment:
In order not to change the existing public signature, I've added an additional parameter for the `topicsRegex` and maintained a form without that parameter that passes in null.

The callers of this function all start with some subclass of `ConnectorConfig`, then pull out maxTasks and sinkTopics from it. It might be more elegant to have the signature here be simply `connectorTaskConfigs(String connName, ConnectorConfig config)` and then pull out the necessary contents here rather than making that the caller's responsibility. Any reason not to do that?